.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024, Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Lint
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+-stable"
+    paths-ignore:
+      - '**/*.md'
+      - '.github/**'
+  pull_request:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+-stable"
+    paths-ignore:
+      - '**/*.md'
+      - '.github/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Lint
+        run: |
+          make lint
+      - name: Store raw test results
+        if: ${{ always() }}
```

Comment:
Interesting. I'm just curious why this magic is necessary.

Reply:
Me neither ... (I got it from the go tests workflow)

Reply:
@yash-zededa, @uncleDecart, do you have an idea of whether we need it here? )